### PR TITLE
shared heal ash ritual + more

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
A new ritual is out boys and more!

Shared Damage:
The person in the center will have their brute and burn evenly spread amongst their alive peers (taken in the damage as brute).

Ritual descriptions:
You know, would you believe me if I said I almost added descriptions on the first pass for rituals? Yea, I thought against it until a while ago when people mentioned how some names are just not descriptive enough-- well said!

## How This Contributes To The Skyrat Roleplay Experience
Maybe you want to play as a shaman with the rune that can split the damage-- that's pretty magic-y!
Descriptions make it much nicer.

## Changelog

:cl:
add: added a share damage ash ritual
qol: ash rituals now have descriptions; examine the rune for more details
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/ashwalkers/code/effects/ash_rituals.dm [hunk 1]
```diff
@@ -130,11 +137,12 @@
 
 /datum/ash_ritual/ash_ceremony/ritual_success(obj/effect/ash_rune/success_rune)
 	. = ..()
-	for(var/mob/living/carbon/human/human_target in range(1))
+	for(var/mob/living/carbon/human/human_target in range(2, get_turf(success_rune)))
```
Comment (larentoun) [inline modular_skyrat/modules/ashwalkers/code/effects/ash_rituals.dm:140]: I'm not that experienced, but wouldn't `in range()` check every atom for type? Wouldn't it be more reasonable to use `viewers()` for that?

File: modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm [hunk 2]
```diff
@@ -20,8 +20,9 @@ GLOBAL_LIST_EMPTY(ash_rituals)
 	if(!current_ritual)
 		. += span_notice("<br>There is no selected ritual at this moment-- use the central rune to select a ritual.")
 		return
-	. += span_notice("<br>The current ritual is: [current_ritual.name]<br>")
-	. += span_warning("The required components are as follows:")
+	. += span_notice("<br>The current ritual is: [current_ritual.name]")
```
Comment (larentoun) [inline modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm:23]: `</br>`?

File: modular_skyrat/modules/ashwalkers/code/effects/ash_rituals.dm (hunk 1 above)
Comment (jjpark-kb) [reply modular_skyrat/modules/ashwalkers/code/effects/ash_rituals.dm:140]: in range does check the atoms and the type-- but that is fine. Viewers are going to be specifically things that view the atom, which doesn't really fit what I want.

File: modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm (hunk 2 above)
Comment (jjpark-kb) [reply modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm:23]: Break line, it creates a space between the things.

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by xXPawnStarrXx** [comment]
Does it work on corpses, or do they need to be living?

**Comment by larentoun** [comment]
> Does it work on corpses, or do they need to be living?

I think I saw something about `stat == DEAD`

**Pull request merged**